platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -78,6 +79,12 @@
 	 */
 	public static native void newcontext(boolean p_32_bits);
 
+	/**
+	 * Invoked on the Vulkan thread when the underlying Android surface is created.
+	 * @param p_surface
+	 */
+	public static native void surfacecreated(Surface p_surface);
```

Comment:
The method name should be updated to be camelCase (`surfaceCreated`).

In the native layer, the logic for this method is almost identical to `newcontext`. The two should be merged into one.